go/go.sum

```diff
@@ -848,6 +848,8 @@ github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0/go.mod h1:D/8v3k
 github.com/bytecodealliance/wasmtime-go v0.36.0/go.mod h1:q320gUxqyI8yB+ZqRuaJOEnGkAnHh6WtJjMaT2CW4wI=
 github.com/camunda/zeebe/clients/go/v8 v8.3.1 h1:Z3L69WRdR4anZSYWVwfLImV5Tj20yWhMb4PuLTfWwb0=
 github.com/camunda/zeebe/clients/go/v8 v8.3.1/go.mod h1:KbIhMSX7PO9hDhqwTwuVONYQDxu81YptJQK3OFi8Ass=
```

Comment:
🔧  not a big deal, I think we should add `go mod tidy` to the steps for doing this update, as otherwise old version stay in the sum file unnecessarily. This command will clean it up.

Reply:
@megglos good suggestion! Updated!